pkg/ansible/run.go

```diff
@@ -56,7 +61,31 @@ func Run(flags *aoflags.AnsibleOperatorFlags) {
 		log.Fatal(err)
 	}
 
-	printVersion()
+	name, found := os.LookupEnv(k8sutil.OperatorNameEnvVar)
+	if !found {
+		log.Fatal("OPERATOR_NAME environment variable not set")
```

Comment:
Just a nit but I thought we were using logr for all the operator logs. We're using logrus here.

Reply:
Gotcha. I presume then we need to change this whole file, which is maybe better-done as a follow-up PR?

Reply:
Yep we can change this in a separate PR.